components/recommendation/RecommendationArticle.vue

```diff
@@ -60,4 +53,14 @@ async function copyLink(url: string, event: Event): void {
 a:hover h3 {
   text-decoration: underline;
 }
+p {
+  line-height: 1.6em;
+  max-height: 4.8em;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  overflow-wrap: anywhere;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+}
```

Comment:
Putting all this CSS here instead of using UnoCSS for readability.

All these pieces are needed for clamping the line, so instead of breaking it up into different locations it felt best to colocate it all here.

The `p` tag had an existing `line-height` of `1.6`, so I'm explicitly declaring it here so it makes sense with the `max-height`.

Reply:
Good call